src/net/tls.cc

```diff
@@ -1158,14 +1183,19 @@ class session : public enable_lw_shared_from_this<session> {
         }
         return _in.get().then([this](buf_type buf) {
             _eof |= buf.empty();
+            tlslog.trace("wait_for_input: buf size={} eof={}", buf.size(), _eof);
```

Comment:
I get the desire to printf-debug TLS state machine, but in a real world scenario, since none of this log messages identify the TLS connection in question, they will just be annoying noise once num connections >= 2. I again really don't think a logger is appropriate for this level of support code.